src/artemis/nexus_writing/write_nexus.py

```diff
@@ -162,7 +162,7 @@ def create_detector_parameters(detector_params: DetectorParams) -> Dict:
         "flatfield_applied": "_dectris/flatfield_correction_applied",
         "pixel_mask": "mask",
         "pixel_mask_applied": "_dectris/pixel_mask_applied",
-        "image_size": [detector_pixels.width, detector_pixels.height],  # (fast, slow)
+        "image_size": [detector_pixels.height, detector_pixels.width],  # (fast, slow)
```

Comment:
Should: This comment is now wrong, it should be  `# (slow, fast)` to match the `fast_axis`, `slow_axis` above.